github/http.go

```diff
@@ -38,6 +38,8 @@ var inspectHeaders = []string{
 	"Location",
 	"Link",
 	"Accept",
+	"Cookie",
+	"User-Agent",
```

Comment:
You have changed the format of the HUB_VERBOSE output (you've added additional headers), so this test broke: https://github.com/github/hub/blob/87a31b093bcc30a726b5f09bb4f4b86343659d5d/features/create.feature#L271-L288

Reply:
Thanks, that was for development